docs/advanced_usage.rst

```diff
@@ -485,16 +486,27 @@ the console, and changes can be made as needed.
 uWSGI
 -----
 
-The default configuration of uWSGI applications does not include the
-``--enable-threads`` setting which must be set to ``true`` for the
-tracing library to run.  This is noted in their best practices doc_.
+The tracer and profiler support uWSGI when configured with the following:
+
+- Threads must be enabled.
+- Multiple processes requires `lazy-app <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#lazy-apps>`_ enabled or a `master <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#master>`_ process enabled.
+
+Tracing is not supported when using ``ddtrace-run`` with uWSGI as the tracer needs to be initialized in the worker processes. Use manual configuration instead to enable tracing by adding :ref:`patch_all()<patch_all>` and :ref:`agent configuration<agentconfiguration>` to a WSGI app::
 
-  .. _doc: https://uwsgi-docs.readthedocs.io/en/latest/ThingsToKnow.html
+  from os import environ
+  from ddtrace import patch_all
+  from ddtrace import tracer
 
-Example run command:
 
-``ddtrace-run uwsgi --http :9090 --wsgi-file your_app.py --enable-threads``
+  patch_all()
+  tracer.configure(
+      collect_metrics=environ.get("DD_RUNTIME_METRICS_ENABLED"),
```

Comment:
why you have to use a `DD_` variable as a user here is confusing.

Reply:
Removed